### ISSUE TITLE
Update to Redmine 3+

### ISSUE THREAD

**Issue by BenoitDuffez** [a-lek/redmine_ldap_push#1]
This looks really nice but it doesn't work on Redmine 3.
Any chance you can update this to make it work on recent Redmine versions?

**Comment by a-lek** [comment]
Yes there are, since we have to update our Redmine instance to 3.x :)

Are you in a hurry?

**Comment by BenoitDuffez** [comment]
I am not in a hurry :)
I saw that there is one fork for this project that works on Redmine 3, however it fails at the connection test. I have seen that the plugin tries to connect to 127.0.0.1 even though I have configured another host.
I have added logging in the plugin that confirm the URL is http://my-host.com:389 but the error message still mentions 127.0.0.1.

I don't know Ruby enough to help more (I'm more of a Java/php guy).
